src/main/constraints/108_user_instant_launches.sql

```diff
@@ -0,0 +1,15 @@
+SET search_path = public, pg_catalog;
+
+ALTER TABLE user_instant_launches
+    ADD CONSTRAINT user_instant_launches_user_id_fkey
+    FOREIGN KEY (user_id)
+    REFERENCES users(id) ON DELETE CASCADE;
+
+ALTER TABLE user_instant_launches
+    ADD CONSTRAINT user_instant_launches_added_by_fkey
+    FOREIGN KEY (added_by)
+    REFERENCES users(id);
+
+ALTER TABLE user_instant_launches
+    ADD CONSTRAINT user_instant_launches_version_unique
+    UNIQUE (version);
```

Comment:
This constraint should probably be on both the `version` and `user_id` columns.

Reply:
Fixed. Thanks.